.vale/styles/RedHat/Usage.yml

```
@@ -1,151 +1,357 @@
+  off-load: offload
+  ok: OK
+  omni-channel: omnichannel
+  on demand: on-demand
+  on-boarding: onboarding
+  on-line: online
+  ops: operations
+  orientate: orient
+  parent/child: parent-child
+  pathname: path name
+  PCOMM: Personal Communications
+  phillips screw: Phillips screw
+  phillips screwdriver: Phillips screwdriver
+  PL/1: PL/I
+  planar board: system board
+  Plug-and-Play: plug-and-play
+  plugin: plug-in
+  pm: PM
+  pod cast|Podcast|Pod Cast: podcast
+  pop-up list: list
+  pop-up menu: menu
+  pop-up window: window
+  pops up: opens
+  power up: power on|turn on
+  poweroff: power-off
+  poweron: power-on
+  practise: practice
+  preventative: preventive
+  print friendly: printer friendly
+  program product: licensed program
+  programmed operator: program operator
+  Programming Language/I: PL/I
+  pseudo code|pseudo-code: pseudocode
+  pub/sub|publish-subscribe: publish/subscribe
+  pull down: pull-down
+  pulldown: pull-down
+  read-write: read/write
+  recreate: re-create
+  requestor: requester
+  right-justified: right-aligned
+  right-justify: right-align
+  right-most: rightmost
+  round-robin task dispatching: round-robin scheduling
+  run-time: run time|runtime|runtime
+  schemata: schemas
+  screencap: screen capture
+  second name: surname
+  segregate: separate
+  segregation: separation|segmentation
+  serverside: server-side|server side
+  set-up: setup
+  short cut: shortcut
+  shortcut key: keyboard shortcut
+  shortcut menu: menu
+  sign into: sign in to
+  sign off of: sign off from
+  signoff: sign-off
+  signon: sign on to
+  single signon: single sign-on
+  slave: worker|child|helper|replica|secondary
+  smartcard: smart card
+  smartpaper: smart paper
+  sockets interface: socket interface
+  socksified: SOCKS-enabled
+  soft boot: restart the system
+  soft copy: softcopy
+  spawn: create
+  spend: expense|expenditure|outlay
+  standalone: stand-alone
+  synch: sync|synchronize
+  syncpoint: sync point
+  system engineer: systems engineer
+  tablespace: table space
+  tap on: tap
+  TCP/IP network: Internet Protocol network|IP network
+  teleprocessing line: telecommunication line|transmission line
+  testcase: test case
+  third-party software vendor: independent software vendor
+  thumb drive: USB flash drive
+  time box: timebox
+  time line: timeline
+  time stamp: timestamp
+  time-out: timeout|time out
+  timeslice: time slice
+  timezone: time zone
+  tool bar: toolbar
+  tool box: toolbox
+  tool kit: toolkit
+  tool tip: tooltip
+  ToolTip: tooltip
+  totalled: totaled
+  touch base: contact|communicate
+  trade-off|trade off: tradeoff
+  trouble shoot: troubleshoot
+  try and: try to
+  twisty: twistie
+  TY-RAP: cable tie
+  un-install: uninstall
+  unallocate: deallocate
+  uncompilation: decompilation
+  uncompile: decompile
+  underbar: underscore
+  unencrypt: decrypt
+  upgradable: upgradeable
+  USB thumb drive: USB flash drive
+  useable: usable
+  vanilla: un-customized
```

Not sure about _un-customized_  based on the description [here](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#do-not-use-vanilla-adjective-noun)
```suggestion
  vanilla:  not customized
```